pinot-tools/src/main/java/com/linkedin/pinot/tools/admin/command/MoveReplicaGroup.java

```
@@ -0,0 +1,434 @@
+  private String zkHost;
+
+  @Option(name = "-zkPath", aliases = {"--cluster", "-c"}, required = true, metaVar = "<string>", usage = "Zookeeper cluster path(Ex: /pinot")
+  private String zkPath;
+
+  @Option(name = "-exec", required = false, metaVar = "<boolean>", usage = "Execute replica group move")
+  private boolean exec = false;
+
+  @Option(name = "-help", required = false, aliases = { "-h", "--h", "--help"}, metaVar = "<boolean>", usage = "Prints help")
+  private boolean help = false;
+
+
+  private ZKHelixAdmin helix;
+  private PinotZKChanger zkChanger;
+
+  @Override
+  public boolean getHelp() {
+    return help;
+  }
+
+  @Override
+  public String getName() {
+    return "MoveReplicaGroup";
+  }
+
+  public String description() {
+    return "Move complete set of segment replica from source servers to tagged servers in cluster";
+  }
+
+  public String toString() {
+    String retString = "MoveReplicaGroup -srcHostsFile " + srcHostsFile + " -tableName " +
+        tableName + " -zkHost " + zkHost + " -zkPath " + zkPath +
+        (exec ? " -exec" : "");
+    return retString;
+  }
+
+  @Override
+  public void cleanup() {
+
+  }
+
+  public boolean execute()
+      throws IOException, JSONException, InterruptedException {
+    validateParams();
+
+    zkChanger = new PinotZKChanger(zkHost, zkPath);
+    this.helix = zkChanger.getHelixAdmin();
+
+    if (! isExistingTable(tableName)) {
+      LOGGER.error("Table {} does not exist", tableName);
+    }
+    List<String> srcHosts = readHostsFromFile(srcHostsFile);
+    if (srcHosts.isEmpty()) {
+      LOGGER.error("Empty list of servers. Nothing to do");
+      // this is not process error but most likely usage error
+      // exiting with status 1 so that scripts can catch this
+      System.exit(1);
+    }
+
+    LOGGER.info("Source hosts: {}", srcHosts);
+    String serverTenant = getServerTenantName(tableName) + "_OFFLINE";
```

ah.

maybe throw an error if we already have _OFFLINE or _REALTIME ?

Better yet, ask the user to specify. I can imaging needing to move LLC segments